src/functions-and-arrays.js

```diff
@@ -1,19 +1,47 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(num1, num2) {
+
+  
+  if (num1 > num2) {
+    return num1
+  } else {
+      return num2
+    }
+}
 
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
-
+function findLongestWord(arr) {
+
+  let word = 0;
+  let finder = 0;
+  if (arr.length === 0) {
+    return null
+  } else { 
+  for (let i = 0; i < arr.length; i++) {
+    if (word.length < arr[i].length) {
+      word = arr[i].length;
+      finder = i;
+      }
+    }
+    return arr[finder]
+  }
+}
```

Comment:
you also do not need the `finder`, look:
```
for (let i = 0; i < arr.length; i++) {
    if (word.length < arr[i].length) {
      word = arr[i];
      }
    }
    return word
  }
```

Also, you don't need to do an else if, you can save yourself a few lines by just doing an if and then running a loop after that. If the first 'if' is fulfilled, the function will return and stop anyway.